include/simdjson/parsedjson.h

```diff
@@ -12,6 +12,7 @@
 #include "simdjson/portability.h"
 
 #define JSONVALUEMASK 0xFFFFFFFFFFFFFF
+#define JSONSTRINGLENGTHMASK 0xFFFFFFFF
```

Comment:
is it used in this PR anywhere? Perhaps it should be in `get_string_length` ?

Reply:
I don’t think we need it... since we want to support 4GB strings.

I think this is leftover from your code that should be eliminated.

Reply:
Fixed in my last commit.